tests/test_cylindrical_geometry.py

```diff
@@ -32,11 +29,10 @@ def test_cylinder_has_property_values_it_was_created_with(nexus_wrapper):
     assert cylinder.radius == approx(radius)
     assert cylinder.height == approx(height)
     assert cylinder.units == units
-    assert cylinder.geometry_str == "Cylinder"
 
 
-def test_cylinder_units_returns_str_if_bytes_in_file(nexus_wrapper):
-    component = add_component_to_file(nexus_wrapper)
+def test_cylinder_units_returns_str_if_bytes_in_file():
```

Comment:
"in_file" doesn't make sense any more

Reply:
I'm going to remove this test completely i think. I don't think there will be a situation where we will have (or want) a byte-string there, as we are dealing with JSON I/O now, and im not sure byte strings are supported in JSON